### ISSUE TITLE
[Bug] Error when checking canViewField from a console request

### ISSUE THREAD

**Issue by tomdavies** [thejoshsmith/craft-fab-permissions#11]
Hey, I'm seeing errors when using [Feed Me](https://github.com/craftcms/feed-me) to import entries on a site that I'm trying out this plugin on, as the plugin expects an instance of `craft\web\User` when for a console request `$user` will instead be a`craft\console\User`:

```
thejoshsmith\fabpermissions\services\Fab::canViewField() must be an instance of craft\web\User, instance of craft\console\User given, called in /app/vendor/thejoshsmith/craft-fab-permissions/src/services/Fields.php on line 32 - Fab.php: 98
```

**Comment by thejoshsmith** [comment]
Hey @tomdavies, thanks for reporting this. Could you please give me a few more details so I can replicate? 

I've done the following so far:

- Installed the plugin on a fresh project.
- Installed Feed Me.
- Set view/edit permissions on an assets field, text field and related field.
- Mapped a dummy JSON feed to each of these fields.
- Ran the import on CLI using `./craft feed-me/feeds/run --id=1`.

The feed imported fine for me without triggering the error, so any setup details you can give me would be really helpful. Also—Which version of the plugin are you using?

Cheers,
Josh

**Comment by thejoshsmith** [comment]
Disregard the above @tomdavies: I can now replicate!

**Comment by thejoshsmith** [comment]
@tomdavies this should now be resolved in release `1.2.2`. Please let me know if you're still experiencing issues.

**Comment by tomdavies** [comment]
@thejoshsmith awesome, many thanks for the quick fix
